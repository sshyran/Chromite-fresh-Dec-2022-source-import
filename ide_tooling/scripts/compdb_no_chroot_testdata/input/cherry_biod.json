[
  {
    "command": "armv7a-cros-linux-gnueabihf-clang++ -MMD -MF obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include -I../../../../../../../../../../mnt/host/source/src/platform2 -I/build/cherry/usr/include -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/build/cherry -fPIE -DUSE_RTTI_FOR_TYPE_TAGS -pthread -Wno-unused-local-typedefs -DBASE_VER=959929 -pthread -I/build/cherry/usr/include/chromeos -I/build/cherry/usr/include/libusb-1.0 -I/build/cherry/usr/include/libchrome -I/build/cherry/usr/include/perfetto -I/build/cherry/usr/include/glib-2.0 -I/build/cherry/usr/lib/glib-2.0/include -I/build/cherry/usr/include/nss -I/build/cherry/usr/include/nspr -I/build/cherry/usr/include/dbus-1.0 -I/build/cherry/usr/lib/dbus-1.0/include -I/build/cherry/usr/include/power_manager-client -std=gnu++17 -DVCSID=\\\"9999-ff3e7cae478977ce08e4fcc57d3991c5dee4a9fc\\\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c ../../../../../../../../../../mnt/host/source/src/platform2/biod/crypto_init/bio_crypto_init.cc -o obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o",
    "directory": "/build/cherry/tmp/portage/chromeos-base/biod-9999/work/build/out/Default",
    "file": "../../../../../../../../../../mnt/host/source/src/platform2/biod/crypto_init/bio_crypto_init.cc",
    "output": "obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o"
  },
  {
    "command": "armv7a-cros-linux-gnueabihf-clang++ -MMD -MF obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include -I../../../../../../../../../../mnt/host/source/src/platform2 -I/build/cherry/usr/include -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/build/cherry -fPIE -DUSE_RTTI_FOR_TYPE_TAGS -pthread -Wno-unused-local-typedefs -DBASE_VER=959929 -pthread -I/build/cherry/usr/include/chromeos -I/build/cherry/usr/include/libusb-1.0 -I/build/cherry/usr/include/libchrome -I/build/cherry/usr/include/perfetto -I/build/cherry/usr/include/glib-2.0 -I/build/cherry/usr/lib/glib-2.0/include -I/build/cherry/usr/include/nss -I/build/cherry/usr/include/nspr -I/build/cherry/usr/include/dbus-1.0 -I/build/cherry/usr/lib/dbus-1.0/include -I/build/cherry/usr/include/power_manager-client -std=gnu++17 -DVCSID=\\\"9999-ff3e7cae478977ce08e4fcc57d3991c5dee4a9fc\\\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c ../../../../../../../../../../mnt/host/source/src/platform2/biod/tools/bio_crypto_init_main.cc -o obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o",
    "directory": "/build/cherry/tmp/portage/chromeos-base/biod-9999/work/build/out/Default",
    "file": "../../../../../../../../../../mnt/host/source/src/platform2/biod/tools/bio_crypto_init_main.cc",
    "output": "obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o"
  }
]
