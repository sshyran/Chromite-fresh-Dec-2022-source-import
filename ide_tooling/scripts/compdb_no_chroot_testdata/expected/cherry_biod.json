[
  {
    "command": "clang++ --target=arm -MMD -MF obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include -I/usr/local/google/home/oka/os2/src/platform2 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/usr/local/google/home/oka/os2/chroot/build/cherry -fPIE -DUSE_RTTI_FOR_TYPE_TAGS -pthread -Wno-unused-local-typedefs -DBASE_VER=959929 -pthread -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/chromeos -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/libusb-1.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/libchrome -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/perfetto -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/glib-2.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/lib/glib-2.0/include -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/nss -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/nspr -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/dbus-1.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/lib/dbus-1.0/include -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/power_manager-client -std=gnu++17 -DVCSID=\\\"9999-ff3e7cae478977ce08e4fcc57d3991c5dee4a9fc\\\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c /usr/local/google/home/oka/os2/src/platform2/biod/crypto_init/bio_crypto_init.cc -o obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o -stdlib=libc++",
    "directory": "/usr/local/google/home/oka/os2/chroot/build/cherry/tmp/portage/chromeos-base/biod-9999/work/build/out/Default",
    "file": "/usr/local/google/home/oka/os2/src/platform2/biod/crypto_init/bio_crypto_init.cc",
    "output": "obj/biod/crypto_init/bio_crypto_init.bio_crypto_init.o"
  },
  {
    "command": "clang++ --target=arm -MMD -MF obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include -I/usr/local/google/home/oka/os2/src/platform2 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/usr/local/google/home/oka/os2/chroot/build/cherry -fPIE -DUSE_RTTI_FOR_TYPE_TAGS -pthread -Wno-unused-local-typedefs -DBASE_VER=959929 -pthread -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/chromeos -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/libusb-1.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/libchrome -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/perfetto -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/glib-2.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/lib/glib-2.0/include -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/nss -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/nspr -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/dbus-1.0 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/lib/dbus-1.0/include -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include/power_manager-client -std=gnu++17 -DVCSID=\\\"9999-ff3e7cae478977ce08e4fcc57d3991c5dee4a9fc\\\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c /usr/local/google/home/oka/os2/src/platform2/biod/tools/bio_crypto_init_main.cc -o obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o -stdlib=libc++",
    "directory": "/usr/local/google/home/oka/os2/chroot/build/cherry/tmp/portage/chromeos-base/biod-9999/work/build/out/Default",
    "file": "/usr/local/google/home/oka/os2/src/platform2/biod/tools/bio_crypto_init_main.cc",
    "output": "obj/biod/tools/bio_crypto_init.bio_crypto_init_main.o"
  }
]
